src/styles/styles.scss

```diff
@@ -37,6 +36,62 @@ html, body {
   height: 100%;
 }
 
+.index_bottom_container {
+  background-color: #FEF2F2;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.side_left {
+  flex: 60%;
+  padding: 60px;
+}
+
+.side_image {
+  flex: 40%;
+  padding: 60px;
+}
+
+.side_left h3 {
+  font-family: $subheading-font;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 28px;
+  display: flex;
+  align-items: center;
+}
+
+.side_left p {
+  font-family: $subheading-font;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 19px;
+  display: flex;
+  align-items: center;
+}
+
+button {
+  background-color: #FAB7C8;
+  padding: 10px 27px;
+  text-align: center;
+  font-size: 18px;
+}
+
+a {
+  color: #16162C;
+  text-decoration: none;
+  font-family: Raleway;
+
+}
+
+.menu_image {
+  // background-color: black;
+  width: 100%;
+  height:250px;
+  // padding: 20px;
```

Comment:
please remove any commented out styles, if they are not needed we don't need to keep them